DMI DOOH ID Mapping.csv

```diff
@@ -31,7 +31,7 @@ STATIONS,1.6,1,transit
 STATIONS_RAILWAY,1.6.1,106,transit.train_stations
 STATIONS_RAILWAY_ENTRANCES,1.6.1.1,106,transit.train_stations
 STATIONS_RAILWAY_PLATFORMS,1.6.1.2,10602,transit.train_stations.platform
-STATIONS_SUBWAY,1.6.2,10501,transit.subway.subway train
+STATIONS_SUBWAY,1.6.2,10501,transit.subway
```

Comment:
This change makes sense, but if we fix the category, we should also fix the Numeric ID to 105